Sources/Sentry/SentryUIApplication.m

```diff
@@ -10,25 +10,29 @@
 
 @implementation SentryUIApplication {
     UIApplicationState appState;
+    SentryNSNotificationCenterWrapper *_notificationCenterWrapper;
+    SentryDispatchQueueWrapper *_dispatchQueueWrapper;
```

Comment:
`m`: Any reason why this is stored in an interface extension as properties?

Reply:
No real reason. It was like that. Good point. I changed it.